### PR TITLE
Completed css-positioning exercise.

### PR DESCRIPTION
**Pull request by ChorongShin**
Screen Shot 2022-01-28 at 10 45 14 AM
Screen Shot 2022-01-28 at 11 01 08 AM
Screen Shot 2022-01-28 at 11 05 24 AM

### CODE REVIEW

File: css-positioning/absolute.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+  position: absolute;
+  top: 160px;
```
Comment (codingcodymiller) [inline css-positioning/absolute.css:33]: 😱 😱 😱 😱 

This is a **_big_** movement. I think you could reduce how much you are moving this element by approaching it using a different box offset property

File: css-positioning/absolute.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  width: 300px;
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgb(0 0 0 / 75%));
+  position: absolute;
+  top: 200px;
```
Comment (codingcodymiller) [inline css-positioning/absolute.css:40]: Your overlay isn't visible at all 🤔

File: css-positioning/fixed.css [hunk 3]
```diff
@@ -0,0 +1,12 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+  text-align: center;
+  position: fixed;
+  left: 400px;
```
Comment (codingcodymiller) [inline css-positioning/fixed.css:9]: 😱 😱 😱 😱 

Is this to center the text? You can just set the element to be the full width of its containing block, it already has `text-align: center` applied to it.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by ChorongShin** [comment]
Screen Shot 2022-01-28 at 11 46 38 AM
Screen Shot 2022-01-28 at 11 49 15 AM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**